docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -122,63 +122,188 @@ class Sidebar extends React.Component<any, any> {
     })
 */
 
+  getActiveCategoryIndex(at: String, sections: ShorthandValue<any>[]) {
+    for (let i = 0; i < sections.length; i++) {
+      const category = sections[i]
+      if (!('items' in category)) {
+        continue
+      }
+      if (category.items.filter(c => 'title' in c).some(e => e.title.to === at)) {
+        return i
+      }
+    }
+    return -1
+  }
+
+  getActiveTreeItem(at: String, sections: ShorthandValue<any>[]) {
+    for (let i = 0; i < sections.length; i++) {
+      const category = sections[i]
+      if (!('items' in category)) {
+        continue
+      }
+      for (let j = 0; j < category.items.length; j++) {
+        const item = category.items[j]
+        if (!('title' in item)) {
+          continue
+        }
+        if (item.title.to === at) {
+          return item
+        }
+      }
+    }
+    return -1
+  }
+
+  highlighCurrentTreeItem(at: String, sections: ShorthandValue<any>[]) {
```

Comment:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md

There are `activeClassName` and `activeStyle` on `NavLink`. So, we can use:

```jsx
title: { as: NavLink, activeClassName: 'active', content: 'Shorthand Props', to: '/shorthand-props' },
```
```js
const treeItemStyles: ICSSInJSStyle = {
  '& .active': { color: 'red' }
}
```

Reply:
I can't get this to work. Should it highlight the links in red color?